packages/nextra-theme-docs/src/misc/default.config.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export default {
-  github: 'https://github.com/shuding/nextra',
```

Comment:
Also, should I remove the default option here completely? This way, you can choose if you want a button to your repository or not.

Reply:
Sounds great!